vmaas_sync/advisory_sync.go

```diff
@@ -223,7 +223,7 @@ func storeAdvisories(data map[string]vmaas.ErrataResponseErrataList) error {
 		return errors.WithMessage(err, "Parsing advisories")
 	}
 
-	if advisories == nil || len(advisories) == 0 {
```

Comment:
The change looks good, based on return type, the parseAdvisories should never return nil.

However, this whole file does not look good, `parseAdvisories` returns slice and error but when there is an error in `parseAdvisories` it returns `nil, err` which does not match the type. First of all, we need to turn on linter for this file and then we can make such changes

Reply:
As for the change - both nil and empty slice have` len() == 0` that's not a problem.

As for returning `nil` in `parseAdvisories()` - it's also ok, slice can have nil value (uninitialized slice) and in 99% cases you don't have to take special care about it (you can append to and iterate over it just fine).